modules/ROOT/pages/dataweave-system-properties.adoc

```diff
@@ -60,97 +39,95 @@ output application/dw
 ----
 * In DataWeave 2.3.0, the expression returns `PT8952H`.
 * In DataWeave 2.2.0, the expression returns `P1Y8D`.
+* Type: `Boolean`
+* Default: `false`
+Available for language levels: 2.3, 2.4
 
 
+//com.mulesoft.dw.decode_binaries_with_writer_encoding//
+|`com.mulesoft.dw.decode_binaries_with_writer_encoding`|
+If set, then binary values in json are written using JsonWriter encoding, otherwise it will use either the encoding defined in the value schema or the default one.
```

Comment:
```suggestion
DataWeave writes binary values in JSON using the JSON writer (`JsonWriter`) encoding. Otherwise, DataWeave uses  the encoding defined in the value schema or the default encoding.
```